src/main/resources/db/migration/V2_05__fix_missing_application_reference_defect.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE visit SET application_reference = reference
```

Comment:
Assuming it doesn't matter that these application references won't have the new format?

Reply:
Well the application reference is the same as the reference as it's a one to one, but it does not really matter that they are in a different format the new format